build.sbt

```diff
@@ -13,6 +13,11 @@ scalaVersion := "2.11.12"
 
 val sparkVersion = "2.4.3"
 
+// required for isolation-forest
```

Comment:
The isolation forest artifacts should now be available in Maven Central.
 
https://oss.sonatype.org/#nexus-search;quick~com.linkedin.isolation-forest